lib/y2lan_restart_common.pm

```diff
@@ -105,16 +138,34 @@ sub check_network_status {
     type_string "\n\n";                                               # make space for better readability of the console
 }
 
+=head2 verify_network_configuration
+
+ verify_network_configuration([$fn], [$dev_name], [$expected_status], [$workaround], [$no_network_check]);
+
```

Comment:
I think in this case is important to describe the parameters, in particular `$fn`, which is basically a reference to a function with the action to be performed.